comma.go

```diff
@@ -32,12 +28,28 @@ func Comma(v int64) string {
 		j--
 	}
 	parts[j] = strconv.Itoa(int(v))
-	return sign + strings.Join(parts[j:len(parts)], ",")
+	return sign + strings.Join(parts[j:len(parts)], separator)
+}
+
+// Comma produces a string form of the given number in base 10 with
+// commas after every three orders of magnitude.
+//
+// e.g. Comma(834142) -> 834,142
+func (h *EnglishHumanizer) Comma(v int64) string {
```

Comment:
IMHO, not breaking compatibility is critical. But we could add i18n-friendly func aliases :
```
func (h *EnglishHumanizer) Comma(v int64) string {
  return h.DecimalSeparator(v)
}
func (h *EnglishHumanizer) DecimalSeparator(v int64) string {
```